charts/caasperli/templates/ingress.yaml

```diff
@@ -43,19 +30,12 @@ spec:
         paths:
           {{- range .paths }}
           - path: {{ .path }}
-            {{- if and .pathType (semverCompare ">=1.18-0" $.Capabilities.KubeVersion.GitVersion) }}
```

Comment:
as with `ingress.className` we might only want to remove the second bit of the `and`, i'd have to check if pathType is indeed optional in k8s tho